cairo-sys-rs/build.rs

```diff
@@ -14,6 +15,7 @@ fn main() {
             process::exit(1);
         }
     } else {
+        macos_maybe_find_lib_ffi();
```

Comment:
It'd be better to call this function only on macOS instead of making the condition inside the function.

Reply:
that was how I'd written it originally but I thought it might be better to not clutter up the call-site with the cfg annotation. happy to make this change, though.